test/accessibility/apps/dashboard.ts

```diff
@@ -49,9 +58,114 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
+
+    it('add a saved search', async () => {
+      await dashboardAddPanel.addSavedSearch('[Flights] Flight Log');
+      await a11y.testAppSnapshot();
+    });
+
+    it('save the dashboard', async () => {
+      await PageObjects.dashboard.saveDashboard(dashboardName);
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open Edit mode', async () => {
+      await PageObjects.dashboard.switchToEditMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('open options menu', async () => {
+      await PageObjects.dashboard.openOptions();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Should be able to hide panel titles', async () => {
+      await testSubjects.click('dashboardPanelTitlesCheckbox');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Should be able display panels without margins', async () => {
+      await testSubjects.click('dashboardMarginsCheckbox');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open add panel', async () => {
+      await dashboardAddPanel.clickOpenAddPanel();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Add one more saved object to cancel it', async () => {
+      await testSubjects.click('savedObjectTitle[Flights]-Average-Ticket-Price');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Close add panel', async () => {
+      await dashboardAddPanel.closeAddPanel();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Exit out of edit mode', async () => {
+      await PageObjects.dashboard.clickCancelOutOfEditMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Discard changes', async () => {
+      await PageObjects.common.clickConfirmOnModal();
+      await PageObjects.dashboard.getIsInViewMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Test full screen', async () => {
+      await PageObjects.dashboard.clickFullScreenMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Exit out of full screen mode', async () => {
+      await PageObjects.dashboard.exitFullScreenMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Make a clone of the dashboard', async () => {
+      await PageObjects.dashboard.clickClone();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Confirm clone with *copy* appended', async () => {
+      await PageObjects.dashboard.confirmClone();
+      await a11y.testAppSnapshot();
+    });
+
     it('Dashboard listing table', async () => {
       await PageObjects.dashboard.gotoDashboardLandingPage();
       await a11y.testAppSnapshot();
     });
+
+    it('Delete a11y clone dashboard', async () => {
+      await listingTable.searchForItemWithName(clonedDashboardName);
+      await listingTable.checkListingSelectAllCheckbox();
+      await listingTable.clickDeleteSelected();
+      await a11y.testAppSnapshot();
+      await PageObjects.common.clickConfirmOnModal();
+      await listingTable.searchForItemWithName('');
+    });
+
+    it('Open flight dashboard', async () => {
```

Comment:
Well, try as I might I can't resolve the issue with a quick fix.

Fixing real problem is out of scope for right now so I added a comment to the ticket (#38980) tracking the solution.

```suggestion
	// Blocked by https://github.com/elastic/kibana/issues/38980
    it.skip('Open flight dashboard', async () => {
```